packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -1,8 +1,9 @@
-import classNames from 'classnames';
 import React from 'react';
+import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import { Formik, Field, Form } from 'formik';
 import { HintBox, Icon, Input, Text } from '@deriv/components';
+import { useExchangeRate } from '@deriv/hooks';
```

Comment:
@niloofar-deriv Shoudn't use `useMarketRate` everywhere in P2P instead of `useExchangeRate`? 🤔

Reply:
No @farzin-deriv we should not use it everywhere